libkbfs/md_journal.go

```diff
@@ -479,15 +490,19 @@ func (j *mdJournal) convertToBranch(
 		}
 		mdsToRemove = append(mdsToRemove, newID)
 
-		err = tempJournal.append(brmd.RevisionNumber(), newID)
+		// No need to try and preserve unknown fields from the
```

Comment:
I don't really follow the comment here.  Why is it that if fields in the journal entry depend on "changed parts of the MD", they shouldn't be preserved on a conversion?  Unknown fields in the MD itself are being preserved, by virtue of the copy methods, right?

Overall though I don't think it really matters. Seems safer to preserve them, but I'm ok either way.